### PR TITLE
Added fields (not functionality yet) to Profile.java

### PR DESCRIPTION
**Pull request by ghost**
Added fields to Profile.java so that in the future, you will be able to enter new data for your profile.
Faraaz will work on the saving logic.
Added a calendar javascript file so you can click on the calendar
for dates, imported the script into main.html.
I added a simple place-holder toString() method to both Employment.java
and Enrollment.java that can be changed later to actually mean something.

### CODE REVIEW

File: app/models/Profile.java [hunk 1]
```diff
@@ -12,58 +12,88 @@
...
+  public String gender; // The user's gender: female or male
   public String locale; // The user's locale (ISO Language Code and ISO Country
-  
+
   @OneToOne
   public User significant_other; // The user's significant other
   public Date anniversary; // date of anniversary
-  public String website; // The URL of the user's personal website
-  
+  //public String website; // The URL of the user's personal website
```
Comment (amshali) [inline app/models/Profile.java]: If we don't want this just remove it. Don't commit commented codes.

File: app/models/Profile.java [hunk 2]
```diff
@@ -12,58 +12,88 @@
...
+
   public String bio; // The user's biography
-  public Date birthday; // The user's birthday
-  
+  public String interested_in; //genders the user is intersted in: Male, Female, Both, Neither
```
Comment (amshali) [inline app/models/Profile.java]: interestedIn 
naming conventions

File: app/models/Profile.java [hunk 3]
```diff
@@ -12,58 +12,88 @@
...
+  {
+  	return !(relationship_status.equals("Single") || relationship_status.equals("It's complicated")
+  		|| relationship_status.equals("Widowed") || relationship_status.equals("Separated") || relationship_status.equals("Divorced"));
+  }
+
   public Profile(User owner) {
 
     this.anniversary = null;
-    this.bio = "";
+    this.bio = null;
```
Comment (amshali) [inline app/models/Profile.java]: Why null and not ""?

File: app/models/Profile.java [hunk 4]
```diff
@@ -12,58 +12,88 @@
...
     this.owner = owner;
     this.political = null;
-    this.quotes = "";
+    this.quotes = null;
     this.significant_other = null;
-    this.religion = "";
+    this.religion = null;
+
     this.education = new ArrayList<Enrollment>();
+    /*Enrollment e = new Enrollment();
```
Comment (amshali) [inline app/models/Profile.java]: Please remove these comments. They just make the code looks ugly.

File: app/models/Profile.java (hunk 2 above)
Comment (ghost) [inline app/models/Profile.java]: Facebook API calls it that:
interested_in     :  The genders the user is interested in  :     array containing strings

Should I still rename it?

Comment (amshali) [inline app/models/Profile.java]: Yes, I think we better stick to our naming convention. It will have the same meaning.

Comment (ghost) [inline app/models/Profile.java]: I changed the variable names to camel case and updated the commit range for my merge request. 
Is that what I am supposed to do if I make changes to a commit, but it hasn't been merged yet?

**Comment by amshali** [comment]
I ran your code and I get these exceptions:
(Please fix and recommit and push)

application.calendar.jpg action not found

Action not found
Action application.calendar.jpg could not be found. Error raised is Controller controllers.application.calendar not found

play.exceptions.ActionNotFoundException: Action application.calendar.jpg not found
    at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:590)
    at play.mvc.ActionInvoker.resolve(ActionInvoker.java:85)
    at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: Controller controllers.application.calendar not found
    ... 3 more
07:54:09,806 ERROR ~ 

@69j83jl97
application.prev.gif action not found

Action not found
Action application.prev.gif could not be found. Error raised is Controller controllers.application.prev not found

play.exceptions.ActionNotFoundException: Action application.prev.gif not found
    at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:590)
    at play.mvc.ActionInvoker.resolve(ActionInvoker.java:85)
    at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: Controller controllers.application.prev not found
    ... 3 more
07:54:09,821 ERROR ~ 

@69j83jl98
application.next.gif action not found

Action not found
Action application.next.gif could not be found. Error raised is Controller controllers.application.next not found

play.exceptions.ActionNotFoundException: Action application.next.gif not found
    at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:590)
    at play.mvc.ActionInvoker.resolve(ActionInvoker.java:85)
    at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: Controller controllers.application.next not found
    ... 3 more

**Comment by ghost** [comment]
Was this when you went on the About Me page? Or when you just brought up OpenBook?
I am not getting this error.

**Comment by amshali** [comment]
Maybe you don't get these errors because you have next.gif and prev.gif and calendar.jpg but you have not committed them?

**Comment by ghost** [comment]
They were supposed to come with the javascript application. I will add some images and then reconfigure the routes file to find the images manually.
Did the error come up immediately, or when you went on the "About" page?

**Comment by amshali** [comment]
Yes, when I went to About page and hit edit. 
How are they supposed to come with javascript? you have to have them somewhere right?

**Comment by ghost** [comment]
Okay I am recommiting

**Comment by amshali** [comment]
and push please.

**Comment by amshali** [comment]
Thanks. Please work on save and cancel. 
LGTM.

**Pull request merged**